src/index.html

```diff
@@ -1,13 +1,347 @@
+            </article>
+          </div>
+        </div>
+      </section>
+
+      <section class="categories">
+        <div class="container">
+          <h2 class="section-title section-title--max--width">
+            Browse Nothing products by category
+          </h2>
+
+          <div class="categories__wrapper">
+            <article class="category">
+              <div class="category__photos">
+                <a href="#" class="category__link category__link-wide">
+                  <img
+                    src="images/categories/image-1.png"
+                    alt="parrots on the phone"
+                    class="category__photo"
+                  >
+                </a>
+
+                <a href="#" class="category__link category__link-square">
+                  <img
+                    src="images/categories/image-2.png"
+                    alt="earphones"
+                    class="category__photo"
+                  >
+                </a>
+              </div>
+
+              <h3 classs="category__title">
+                All products
+              </h3>
+            </article>
+
+            <article class="category">
+              <div class="category__photos">
+                <a href="#" class="category__link category__link-square">
+                  <img
+                    src="images/categories/image-3.png"
+                    alt="cell phone"
+                    class="category__photo"
+                  >
+                </a>
+
+                <a href="#" class="category__link category__link-wide">
+                  <img
+                    src="images/categories/image-4.png"
+                    alt="battery charger"
+                    class="category__photo"
+                  >
+                </a>
+              </div>
+
+              <h3 classs="category__title">
+                Accessories
+              </h3>
+            </article>
+
+            <article class="category">
+              <div class="category__photos">
+                <a href="#" class="category__link category__link-wide">
+                  <img
+                    src="images/categories/image-5.png"
+                    alt="cell phone"
+                    class="category__photo"
+                  >
+                </a>
+
+                <a href="#" class="category__link category__link-square">
+                  <img
+                    src="images/categories/image-6.png"
+                    alt="battery charger"
+                    class="category__photo"
+                  >
+                </a>
+              </div>
+
+              <h3 classs="category__title">
+                Accessories
+              </h3>
+            </article>
+          </div>
+
+          <div class="page__button-wraper">
+            <a href="#" class="page__button categories__button">
+              View all
+            </a>
+          </div>
+        </div>
+      </section>
+
+      <section class="image"></section>
```

Comment:
will be good to use self-closed tag like `<section class="image" />`

Reply:
![image](https://github.com/mate-academy/layout_landing-page/assets/140401215/bcae4547-2ce8-4514-8977-f771d26d7dcd)
I got an error, so I left it as it is, maybe there is another solution


and so all the changes were made, I hope that I did not miss any point